packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -793,9 +798,11 @@ function commitRoot(root: FiberRoot, finishedWork: Fiber): void {
       // TODO: Avoid this extra callback by mutating the tracing ref directly,
       // like we do at the beginning of commitRoot. I've opted not to do that
       // here because that code is still in flux.
-      callback = Schedule_tracing_wrap(callback);
+      callback = Scheduler_tracing_wrap(callback);
     }
-    passiveEffectCallbackHandle = schedulePassiveEffects(callback);
+    passiveEffectCallbackHandle = runWithPriority(NormalPriority, () => {
+      return schedulePassiveEffects(callback);
+    });
```

Comment:
When I was reading this code last week I was wondering how you'd be able to mess with the `schedulePassiveEffects` priority. Doesn't seem normally possible... unless you cause a `Sync` update I guess...